gradio-blocks/README.md

```diff
@@ -0,0 +1,114 @@
+# Gradio Blocks Party  🥳
+
+![Untitled Design (1)](https://user-images.githubusercontent.com/81195143/167931943-f31e3361-55e5-4092-b0ad-5f6e85258b21.png)
+
+
+_**Timeline**: May 16th, 2022 - May 31st, 2022_
+
+---
+
+Happy to invite you to the Gradio Blocks Party - a community event in which we will build cool machine learning demos using the new Gradio Blocks feature. Blocks allows you to build web-based demos in a flexible way using the Gradio library. The event will take place from May 16th to 31st. We will be organizing this event on github and the huggingface discord channel.
```

Comment:
Might be good to mention here that there are prizes up for grabs to get people's attention 🏆

Reply:
sounds good, Ill mention the prizes section below

Reply:
updated